config/default/testRunner.conf.php

```diff
@@ -402,9 +402,16 @@
             'reviewLayout' => 'default',
             /**
              * Display section titles or not
+             * (Applies only to `fizzy` layout)
              * @type boolean
              */
-            'displaySectionTitles' => true
+            'displaySectionTitles' => true,
+            /**
+             * Show item title as a tooltip
+             * (Applies only to `fizzy` layout)
+             * @type boolean
+             */
+            'displayItemTooltip' => false
```

Comment:
**nitpick:** I don't see the related migration script. Was it intended?

Reply:
There wasn't any for `reviewLayout` and `displaySectionTitles`, so I didn't do it here either. 
Also, I'm afraid that if I add a migration for these 3 settings now, it will override values set by `taoMexcbt` migration. So I think we shouldn't add it.

(Actually, I wonder how config-setter priority is decided for such cases...)

Reply:
The migration scripts are following the dependencies order. So the taoQtiTest migrations would always play before any consumer extension.
The migration is needed here, even if this is to set the default value. It helps to see all the possibilities in the deployed config file, and it also makes sure the config is properly set, with default values, or with custom values. Otherwise, it may cause weird issues (it is maybe not the case for your feature, but this is a global good practice).
It is a pity that it was forgotten by the previous stories, but please do not let any chance for the "broken window effect" to take place. Better to fix issues as soon as possible to prevent bad habits to sit. Otherwise, the next developer will argue that the previous PR did not contain it, so no need to do... :)